tools/accuracy_checker/accuracy_checker/annotation_converters/criteo_kaggle_dac.py

```diff
@@ -32,10 +33,14 @@ def parameters(cls):
         parameters = super().parameters()
         parameters.update({
             'testing_file': PathField(description="Path to testing file."),
+            "binary": BoolField(optional=True, default=False,
+                                    description="Allows input file in binary mode instead of .npz mode"),
             "batch": NumberField(optional=True, default=128, description="Model batch"),
+            "max_ind_range": NumberField(optional=True, default=0,
```

Comment:
please provide constrains, min_value=1 and value_type=int as you use in mod

Reply:
Fixed